### DISCUSSION TITLE
The current css prefix is el-, hope can customize the prefix so that the element2.x version and the css style of element-plus can coexist without interfering with each other

### DISCUSSION THREAD

**Discussion by liyann** [element-plus/element-plus#1154] [Ideas]
目前前缀是el-，希望可以自定义前缀，这样element2.x版本和element-plus的css样式可以共存，互不干扰

**Comment by zazzaz** [comment]
This is interesting. Just wondering how can a project have both vue2.0 and vue 3.0 installed?

**Comment by liyann** [comment]
In fact,we use single-spa to compose vue2.0 and vue3.0 project, they work well.

**Reply by jw-foss** [reply]
However, despite the fact that v2 and v3 share lots of style rule, because the change of `Vue` we have to make changes for some components, also, the reason why we didn't rename the prefix as `ep` is because we want our users to have less migration effort the better, so that in this case we cannot simply rename the prefix for css classes.

**Reply by zazzaz** [reply]
A global prefix config could solve this problem

**Comment by zazzaz** [comment]
I see.

To provide a global config setting class prefix namespace is a nice feature.

However, we have so many hardcode `el-` class name in our component, it's a large job to do.

**Comment by sxzz** [comment]
Related issue #5374
